windows-release/stage-sbom.yml

```diff
@@ -0,0 +1,53 @@
+jobs:
+- job: SBOM_Files
+  displayName: Create SBOMs for Python binaries
+
+  pool:
+    vmImage: windows-2022
+
+  workspace:
+    clean: all
+
+  strategy:
+    matrix:
+      win32:
+        Name: win32
+      amd64:
+        Name: amd64
+      arm64:
+        Name: arm64
+
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.6 or later'
+    inputs:
+      versionSpec: '>=3.6'
+
+  - template: ./checkout.yml
+
+  - task: DownloadPipelineArtifact@1
+    displayName: 'Download artifact: bin_$(Name)'
+    inputs:
+      artifactName: bin_$(Name)
+      targetPath: $(Build.BinariesDirectory)\bin
+
+  - powershell: >
+      python
+      "$(Build.SourcesDirectory)\sbom.py"
+      "--cpython-source-dir=$(Build.SourcesDirectory)"
+      (gci msi\*\python-*.exe)
```

Comment:
The MSIs are not available until the very end of the build, and only if the MSI stage ran (which is optional).

Perhaps a better place to put this is in the `layout` stages? Those should be platform-specific and package-specific already, so you can pass options to the script directly.

Reply:
I took a stab at this in https://github.com/python/release-tools/pull/100/commits/1d23d2da532a389877b86a6c88ad5f29838bcb26, not certain if this is what you were imagining. Also wasn't sure if `$(Python)` would be available in these contexts and if I'm using `gci` correctly. My Windows knowledge is fairly limited :grimacing: